qa/suites/rgw/cloud-transition/cluster.yaml

```diff
@@ -0,0 +1,9 @@
+roles:
+- [mon.a, osd.0, osd.1, osd.2, mgr.0, client.0, client.1]
+openstack:
+- volumes: # attached to each instance
+    count: 2
+    size: 10 # GB
+overrides:
```

Comment:
My question about the `openstack` and `overrides` tasks here from the dbstore PR applies here as well.

Reply:
Done.